src/wesnoth.cpp

```diff
@@ -941,7 +941,7 @@ static void restart_process(const std::vector<std::string>& commandline)
 #define error_exit(res) return res
 #endif
 
-#ifdef __APPLE__
+#ifdef __IPHONEOS__
```

Comment:
Indeed there is a lot of binaries there, among sources and in directories like `.libs`; probably even totally unused. Thus I can commit an iOS project to mainline. Though... do you want **another** copy of all the dependencies in there? :D